Use autogenerated pgouser password if not set (#2010)
CrunchyData/postgres-operator: docs/content/installation/configuration.md

| `namespace_mode` | dynamic |  | Determines which namespace permissions are assigned to the PostgreSQL Operator using a ClusterRole. Options: `dynamic`, `readonly`, and `disabled` |
| `pgbadgerport` | 10000 | **Required** | Set to configure the default port used to connect to pgbadger. |
| `pgo_add_os_ca_store` | false | **Required** | When true, includes system default certificate authorities. |

> | `pgo_admin_password` | examplepassword |  | Configures the pgo administrator password. If not set, a randomly generated password will be set.|

Reviewer: ```suggestion
| `pgo_admin_password` | examplepassword |  | Configures the pgo administrator password. When blank, a random password is generated. |
```

Author: updated.

| `pgo_add_os_ca_store` | false | **Required** | When true, includes system default certificate authorities. |
| `pgo_admin_password` | examplepassword | **Required** | Configures the pgo administrator password. |
| `pgo_admin_perms` | * | **Required** | Sets the access control rules provided by the PostgreSQL Operator RBAC resources for the PostgreSQL Operator administrative account that is created by this installer. Defaults to allowing all of the permissions, which is represented with the * |

> | `pgo_admin_role_name` | pgoadmin |  | Sets the name of the PostgreSQL Operator role that is utilized for administrative operations performed by the PostgreSQL Operator. If not set, a randomly generated password will be set.|

Reviewer: `pgo_admin_password` was removed from the preflight check. I suspect this is a typo intended for L56?

Author: yep, that's correct. Thanks for catching that. Updated.
